test/EasyCaching.UnitTests/CachingTests/RedisCachingProviderTest.cs

```diff
@@ -27,7 +27,8 @@ protected override IEasyCachingProvider CreateCachingProvider(Action<BaseProvide
                 {
                     options.DBConfig = new RedisDBOptions
                     {
-                        AllowAdmin = true
+                        AllowAdmin = true,
```

Comment:
Add UTs to **RedisCachingProviderWithKeyPrefixTest** class for **RemoveByPrefix**.